abr_control/controllers/path_planners/second_order.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, robot_config, n_timesteps=100, dt=0.001, zeta=2.0,
         self.zeta = zeta
         self.w = w/n_timesteps # gain to converge in the desired time
 
-    def step(self, state, target):
+    def step(self, y, dy, target, w, zeta, dt=0.001, threshold=0.02):
```

Comment:
humm the idea behind state was to keep the format fro input to all path planners the same, to that end maybe we also want a dict at the end `params` instead of specifying them all

Reply:
Switching over to state makes sense to me. Maybe we can just specify the other parameters in the init?

Reply:
yeah i'll draft a change of this and push

Reply:
Are you just updating the path planner of examples too?